ga4gh/datamodel/__init__.py

```diff
@@ -508,6 +529,21 @@ def getParentContainer(self):
         """
         return self._parentContainer
 
+    def _scanDataFiles(self, dataDir, patterns):
```

Comment:
If it's a more natural map to the form of the G2P data, then using ``_scanDataFiles`` is fine I think.